src/data-card-advanced.tsx

```diff
@@ -0,0 +1,463 @@
+
+    const flexDirection = hasAllItens ? 'column' : 'row';
+    const alignItems = hasAllItens ? 'start' : 'center';
+    const marginTop = hasAllItens ? '8px' : '16px';
+    const maxWidth = hasButtonLink && !hasAllItens ? '178px' : '';
+
+    return (
+        <div>
+            <NegativeBox>
+                <Divider />
+            </NegativeBox>
+
+            <div className={styles.actions} style={{flexDirection, alignItems}}>
+                {hasButton && (
+                    <div
+                        aria-hidden="true"
+                        onClick={(event) => event.stopPropagation()}
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        style={{marginTop: '16px', marginRight: '16px', position: 'relative', zIndex: '2'}}
+                    >
+                        {button}
+                    </div>
+                )}
+                <div style={{display: 'flex', flexDirection: 'row', marginTop, flexWrap: 'wrap'}}>
+                    {hasFooterImage && (
+                        <div
+                            style={{marginRight: '16px', zIndex: '0'}}
+                            className={sprinkles({alignItems: 'center', display: 'flex'})}
+                        >
+                            {footerImage}
+                        </div>
+                    )}
+
+                    {hasFooterText && (
+                        <div style={{maxWidth}} className={styles.footerText}>
+                            <Text2 truncate={footerTextLinesMax} regular>
+                                {footerText}
+                            </Text2>
+                        </div>
+                    )}
+                </div>
+                {hasButtonLink && (
+                    <div
+                        aria-hidden="true"
+                        className={sprinkles({
+                            display: 'flex',
+                        })}
+                        onClick={(event) => event.stopPropagation()}
+                        style={{marginTop, position: 'relative', zIndex: '2'}}
+                    >
+                        {buttonLink}
+                    </div>
+                )}
+            </div>
+        </div>
+    );
+};
+
+type slotsTypeof = typeof StackingGroup;
+
+type textAs = 'h1' | 'h2' | 'h3' | 'h4';
+
+interface DataCardAdvancedProps {
+    cardOnPress?: () => void;
+
+    stackingGroup?: RendersNullableElement<typeof StackingGroup>;
+    headline?: string | RendersNullableElement<typeof Tag>;
+    pretitle?: string;
+    pretitleLinesMax?: number;
+    pretitleAs?: textAs;
+    title?: string;
+    titleLinesMax?: number;
+    titleAs?: textAs;
+    subtitle?: string;
+    subtitleLinesMax?: number;
+    description?: string;
+    descriptionLinesMax?: number;
+
+    slots?: Array<RendersNullableElement<slotsTypeof>>;
```

Comment:
why is this an array?
I'd call this extra, to follow the same naming conventions we use in other components

Reply:
because unlike the normal data card, this one will not receive a children but an array of slots which is another PR for not having the possibility of adding other elements besides the slots